packages/views/screens/managers/multiSignature/form/form.test.js

```diff
@@ -2,15 +2,15 @@ import React from 'react';
 import { mount } from 'enzyme';
 import { act } from 'react-dom/test-utils';
 
-import { getTransactionBaseFees, getTransactionFee } from '@transaction/utilities/api';
-import useTransactionFeeCalculation from '@transaction/manage/transactionPriority/useTransactionFeeCalculation';
+import { getTransactionBaseFees, getTransactionFee } from '@transaction/utils/api';
+import useTransactionFeeCalculation from '@transaction/hooks/useTransactionFeeCalculation';
 import { fromRawLsk } from '@token/utilities/lsk';
 import wallets from '@tests/constants/wallets';
 
 import Form, { validateState } from './form';
 
-jest.mock('@transaction/manage/transactionPriority/useTransactionFeeCalculation');
-jest.mock('@transaction/utilities/api');
+jest.mock('@transaction/hooks/useTransactionFeeCalculation');
+jest.mock('@transaction/utils/api');
```

Comment:
@eniolam1000752 This needs to be `@transaction/api` since the current path doesn't exist